### PR TITLE
Update LAYOUT_60_ansi_arrow

### PR DESCRIPTION
**Pull request by mrsendyyk**
## Description

Update `LAYOUT_60_ansi_arrow`

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mrsendyyk** [comment]
Can you help me to request a review from qmk/collaborators @ridingqwerty?

**Comment by mrsendyyk** [comment]
Ccc: @fauxpark @drashna @tzarc @noroadsleft :)

File: layouts/default/60_ansi_arrow/default_60_ansi_arrow/keymap.c [hunk 1]
```diff
@@ -1,67 +1,81 @@
...
+        if (host_keyboard_led_state().num_lock) {
+          rgblight_setrgb(225, 8, 0);
+        }
+     /* Scroll Lock Indicator */
+        if (host_keyboard_led_state().scroll_lock) {
+          rgblight_setrgb(241, 190, 72);
+        }
+   }
+
+   void led_set_user(uint8_t usb_led) {
```
Comment (drashna) [inline layouts/default/60_ansi_arrow/default_60_ansi_arrow/keymap.c:52]: This should be updated to use:
https://docs.qmk.fm/#/custom_quantum_functions?id=led-control


Also, it looks/sounds like you may want to look into the rgb lighting layers stuff.
https://docs.qmk.fm/#/feature_rgblight?id=lighting-layers

File: layouts/default/60_ansi_arrow/readme.md [hunk 2]
```diff
@@ -1,50 +1,76 @@
-# 60_ansi_arrow
-`LAYOUT_60_ansi_arrow`
+# 60_ansi_arrow and Keymap
```
Comment (fauxpark) [inline layouts/default/60_ansi_arrow/readme.md:1]: This is a default layout, so it should be made as simple as possible.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: layouts/default/60_ansi_arrow/readme.md (hunk 2 above)
Comment (mrsendyyk) [reply layouts/default/60_ansi_arrow/readme.md:1]: Ok done.

File: layouts/default/60_ansi_arrow/default_60_ansi_arrow/keymap.c (hunk 1 above)
Comment (mrsendyyk) [reply layouts/default/60_ansi_arrow/default_60_ansi_arrow/keymap.c:52]: Can you suggested the change @drashna?

**Comment by mrsendyyk** [comment]
Sorry @ridingqwerty, i accidentally dismissed your stale review.

File: layouts/default/60_ansi_arrow/readme.md (hunk 2 above)
Comment (mrsendyyk) [reply layouts/default/60_ansi_arrow/readme.md]: Ok done

**Pull request closed without merging**